hazelcast/src/main/java/com/hazelcast/security/permission/ConnectorPermission.java

```diff
@@ -61,6 +62,21 @@ public static ConnectorPermission jdbc(@Nullable String connectionUrl, String ac
         return new ConnectorPermission(JDBC_PREFIX + (connectionUrl == null ? "" : connectionUrl), action);
     }
 
+    /**
+     * @param connectionDescription connection string or data connection name
+     * @param databaseName database name this permission is about, null if permission is granted to all databases
+     * @param collectionName collection name this permission is about, null if permission is granted to all collections
+     * @param action action this permission is about
+     */
+    public static ConnectorPermission mongo(@Nullable String connectionDescription,
+                                            @Nullable String databaseName, @Nullable String collectionName,
+                                            String action) {
+        String db = databaseName == null ? "*" : databaseName;
+        String col = collectionName == null ? "*" : collectionName;
+        String dbCol = db + "/" + col;
+        return new ConnectorPermission(MONGO_PREFIX + connectionDescription + ":" + dbCol, action);
```

Comment:
Do you want to allow `null` to appear in the String? The `connectionDescription` is `@Nullable`.

Reply:
https://github.com/hazelcast/hazelcast/pull/24467/commits/f1c978f3a24f735b10713f0e35535ebb726b1c35